indexer/src/service.rs

```diff
@@ -151,7 +203,71 @@ impl IndexerService {
                                     .encode(&[token.clone()])
                                     .expect("Bad Encoding!");
                                 // TODO: should wrap this in a db transaction.
-                                if let Err(e) = exec.trigger_event("an_event_name", vec![args]) {
+                                if let Err(e) = exec.trigger_event(ReceiptEvent::Log, vec![args]) {
+                                    error!("Event processing failed {:?}", e);
+                                }
+                            }
+                            Receipt::LogData {
+                                id,
+                                ra,
+                                rb,
+                                ptr,
+                                len,
+                                digest,
+                                data,
+                                pc,
+                                is,
+                            } => {
+                                // TODO: might be nice to have Receipt type impl Tokenizable.
+                                let token = Token::Struct(vec![
+                                    id.into_token(),
```

Comment:
Again, https://github.com/FuelLabs/fuel-indexer/issues/58 might avoid some of this boiler plate (but this isn't terrible since it's abstracted away from the user)